docs-source/spring/content/development/cli/_index.md

```diff
@@ -952,11 +969,222 @@ GraalVM Compile Commands
         help for compile-purge   
         [Optional]
 
-        Ask for Help
-            Slack: https://oracledevs.slack.com/archives/C03ALDSV272
-            E-mail: obaas_ww@oracle.com
+    Ask for Help
+        Slack: https://oracledevs.slack.com/archives/C03ALDSV272
+        E-mail: obaas_ww@oracle.com
     ```
 
+### User Management
+
+Manage users allows you to create the platform users and assign the roles that give access permission to operate with the platform.
+
+**User Roles**
+
+* `ROLE_ADMIN`: Users with this role have the access rights as a `ROLE_USER` and the right of `delete` applications. Also, only a user with this role can create new users, search for registered users, change passwords and roles from the other users, and delete users.
+
+* `ROLE_CONFIG_EDITOR`: This role is able to edit the platform configurations. **Currently unused in the environment**.
+
+* `ROLE_USER`: This role is be able to connect to the Admin service, create and list for applications (namespaces); use the [bind](#bind) command; deploy, list and scale workloads (services).
```

Comment:
still has semi-colons where there should be commas, "for" does not make sense, if you are going to add a link to bind, add it to all of them- be consistent, all or none.  Comma after the and.